### ISSUE TITLE
official botloader of Joy-IT OR750i?

### ISSUE THREAD

**Issue by SvenRoederer** [CodeFetch/router-u-boot#13]
Is this the official code of the bootloader of the Joy-IT OR750i?

Asking, because my board is booting, but I'm not able to reach the Recovery website or make it downloading an image from my tftp-server.

**Comment by CodeFetch** [comment]
No, it isn't. There are a lot of additional patches for which I haven't found time to add them, yet.
Hold the reset button for 4 seconds. TFTP server IP address must be 192.168.0.66. The bootloader does a ping check first. If the server is not reachable, it attempts to boot. Please note that you must connect it to the LAN ports as the WAN port is deactivated in the bootloader.
You can also hold the button for 6 seconds in which case it will try to acquire a DHCP lease and get the firmware file via TFTP from the DHCP server's IP address.

But the question is why you can't access the web interface... It should be possible. Do you see the fading light pattern after holding the reset button for 10 seconds? Please note that you must connect the PC to the LAN port before.
